integrasjontjenester/fagsystem-klient-api/src/main/java/no/nav/foreldrepenger/tilbakekreving/fagsystem/klient/dto/SendtoppdragDto.java

```diff
@@ -0,0 +1,4 @@
+package no.nav.foreldrepenger.tilbakekreving.fagsystem.klient.dto;
+
+public record SendtoppdragDto(String saksnummer, Long behandlingId) {
```

Comment:
DTO-en fungerer ikke så bra for oss, siden vi ikke bruker Long for å identifisere behandlingen utenfor k9sak. Vi trenger å ha UUID eller string

Reply:
Det ser ikke ut som noen av feltene brukes, så kan evt erstattes med en Boolean i SamletEksternBehandlnigInfo, og så bruke DTO-en bare i fpsak-klienten?

Reply:
Da kan vi evt også starte bare med en super-enkel implementasjon hvor K9sakKlient bare returnerer 'true' for om det finnes oppdrag, og så impementere faktisk kall til k9-oppdrag når det passer

Reply:
Jeg skal se på å generifisere oppslaget så det virker for begge.